### PR TITLE
Fix i18n, update resource bundles

### PR DESCRIPTION
**Pull request by hieudd**
Fix i18n, update resource bundles

### CODE REVIEW

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties [hunk 1]
```diff
@@ -996,6 +982,8 @@ your_listed_crf_in_the_file = Your listed Event CRF in the file {0} does not exi
 
 your_message_sent_succesfully = Your message has been sent successfully.
 
+your_message_sent_succesfully_to = Your message has been sent successfully {0}.
+
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: This string is not referenced in the code and should be removed.

File: web/src/main/resources/org/akaza/openclinica/i18n/words.properties [hunk 2]
```diff
@@ -1617,10 +1632,16 @@ removed = Removed
 
 repeat = Repeat
 
+execute_rule = Execute Rule
+
 repeat_as_necessary = Repeat<br> As<br> Necessary
 
 repeat_max = Repeat Max
 
+is_shown = is_shown
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/words.properties]: Should this be "is_shown = is shown"?

File: web/src/main/webapp/WEB-INF/jsp/include/studySideInfo.jsp [hunk 3]
```diff
@@ -4,69 +4,70 @@
 
 <fmt:setBundle basename="org.akaza.openclinica.i18n.words" var="resword"/>
 <fmt:setBundle basename="org.akaza.openclinica.i18n.format" var="resformat"/>
+<fmt:setBundle basename="org.akaza.openclinica.i18n.page_messages" var="respage"/>
```
Comment (sdibona) [inline web/src/main/webapp/WEB-INF/jsp/include/studySideInfo.jsp]: respage is not referenced.  should be removed.

File: web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp [hunk 4]
```diff
@@ -15,318 +15,322 @@
...
-
-		</div>
-
-	</div></div></div></div></div></div></div>
+    <table border="0" cellpadding="0" cellspacing="0">
+        <tr id="CRF_infobox_closed" style="display: none;">
+            <td style="padding-top: 3px; padding-left: 6px; width: 90px;" nowrap>
+                <a href="javascript:leftnavExpand('CRF_infobox_closed'); leftnavExpand('CRF_infobox_open');"><img src="images/sidebar_expand.gif" border="0" align="right" hspace="10"></a>
+
+                <b>CRF Info</b>
```
Comment (sdibona) [inline web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp]: If you are removing hardcoded strings in this file, this one should also be added to a properties file.

File: web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp [hunk 5]
```diff
@@ -15,318 +15,322 @@
...
+                        <td valign="bottom">
+                            <table border="0" cellpadding="0" cellspacing="0" width="100">
+                                <tr>
+                                    <td nowrap>
+                                        <div class="tab_BG_h">
+                                            <div class="tab_R_h" style="padding-right: 0px;">
+                                                <div class="tab_L_h" style="padding: 3px 11px 0px 6px; text-align: left;">
+                                                    <a href="javascript:leftnavExpand('CRF_infobox_closed'); leftnavExpand('CRF_infobox_open');"><img src="images/sidebar_collapse.gif" border="0" align="right"></a>
+
+                                                    <b>CRF Info</b>
```
Comment (sdibona) [inline web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp]: If you are removing hardcoded strings in this file, this one should also be added to a properties file.

**Comment by sdibona** [comment]
Hello,

Thank you for the i18n updates!  

A general comment.  When you are adding new strings to the properties file, please put them at the bottom of the file.  It makes it easier for people maintaining translations to find the new strings.

Also, would you please re-create this pull request into the newly created OpenClinica:3.13 branch? We'd like to include it in the next 3.x release.

It's currently open against OpenClinica:master and contains many additional changes for 4.x which we don't want pulled into 3.x. To get around this you will need to create a branch off of 3.13. Then execute 'git cherry-pick' to apply only this commit to your 3.13 branch. Then re-create the pull request. If this is a problem, please let me know and I'll assist with getting these changes into the 3.13 branch.

Thank you,
-Steve DiBona

**Comment by hieudd** [comment]
Hi @sdibona please check and give me your comments.

1. https://github.com/OpenClinica/OpenClinica/pull/737
2. https://github.com/OpenClinica/OpenClinica/pull/738
3. https://github.com/OpenClinica/OpenClinica/pull/739

Thank you very much!

**Pull request closed without merging**